Remotion/ObjectBinding/Web/UI/Controls/BocBooleanValueImplementation/Rendering/BocCheckBoxRenderer.cs

```diff
@@ -207,8 +207,12 @@ private string GetScriptParameters (BocCheckBoxRenderingContext renderingContext
       string script = " ("
                       + checkBox + ", "
                       + label + ", "
-                      + (string.IsNullOrEmpty (renderingContext.Control.TrueDescription) ? "null" : "'" + renderingContext.Control.TrueDescription + "'") + ", "
-                      + (string.IsNullOrEmpty (renderingContext.Control.FalseDescription) ? "null" : "'" + renderingContext.Control.FalseDescription + "'") + ");";
+                      + (renderingContext.Control.TrueDescription.IsEmpty
+                          ? "null"
+                          : "'" + renderingContext.Control.TrueDescription.ToString (WebStringEncoding.HtmlWithTransformedLineBreaks) + "'") + ", "
+                      + (renderingContext.Control.FalseDescription.IsEmpty
+                          ? "null"
+                          : "'" + renderingContext.Control.FalseDescription.ToString (WebStringEncoding.HtmlWithTransformedLineBreaks) + "'") + ");";
```

Comment:
It's a bug that we don't use EscapeClientScript here. Please update (it's mentioned also in other tasks where this oversight was more obvious during task design)